app/views/lists/show.html.erb

```diff
@@ -38,7 +38,7 @@
         <ul class="govuk-summary-list__actions-list gem-c-summary-list__group-actions-list">
           <li class="govuk-summary-list__actions-list-item">
             <%= link_to 'Add links to list',
-              tag_list_list_items_path(@tag, @list),
```

Comment:
Is there any additional code that can be removed here now that we're not using this path? An endpoint in routes.rb or anything else? (I struggle to follow these magic ruby path methods so maybe there is an obvious answer here that I'm missing :-) )